README.md

```diff
@@ -6,149 +6,96 @@
-are available in your distribution's repository.  By default, openshift-ansible
-will not configure extra repositories for testing or staging packages for
-end users.
-
-We recommend using a release branch. We maintain stable branches
-corresponding to upstream Origin releases, e.g.: we guarantee an
-openshift-ansible 3.2 release will fully support an origin
-[1.2 release](https://github.com/openshift/openshift-ansible/tree/release-1.2).
-
-The most recent branch will often receive minor feature backports and
-fixes. Older branches will receive only critical fixes.
-
-In addition to the release branches, the master branch
-[master branch](https://github.com/openshift/openshift-ansible/tree/master)
-tracks our current work **in development** and should be compatible
-with the
-[Origin master branch](https://github.com/openshift/origin/tree/master)
-(code in development).
+### Requirements
 
+#### Ansible
+[Ansible 2.7](https://docs.ansible.com/ansible/2.5/installation_guide/intro_installation.html) 
 
+#### Packages
+```
+pip install -U apache-libcloud~=2.2.1
+pip install google-auth
+yum install -y google-cloud-sdk python2-crypto
+```
 
-**Getting the right openshift-ansible release**
+#### GCP Credentials
 
-Follow this release pattern and you can't go wrong:
+Place your service account key with the name `gce.json` in the root of OpenShift-Ansible.
 
-| Origin/OCP    | OpenShift-Ansible version | openshift-ansible branch |
-| ------------- | ----------------- |----------------------------------|
-| 1.3 / 3.3          | 3.3               | release-1.3 |
-| 1.4 / 3.4          | 3.4               | release-1.4 |
-| 1.5 / 3.5          | 3.5               | release-1.5 |
-| 3.*X*         | 3.*X*             | release-3.x |
+OpenShift developers can find the service account key in the [shared secrets repo](https://github.com/openshift/shared-secrets/blob/master/gce/aos-serviceaccount.json).
 
-If you're running from the openshift-ansible **master branch** we can
-only guarantee compatibility with the newest origin releases **in
-development**. Use a branch corresponding to your origin version if
-you are not running a stable release.
 
+### files_dir (do this once) TODO:REMOVE
 
-## Setup
 
-Install base dependencies:
+All files (bootstrap.ign, gce.json) should be saved to the root of this repo. Tell the playbooks where to find them:
 
-Requirements:
+```
+# Execute this from the root dir
+echo files_dir: $PWD > inventory/dynamic/gcp/group_vars/all/00_default_files_dir.yml
+```
 
-- Ansible >= 2.6.5, Ansible 2.7 is not yet supported and known to fail
-- Jinja >= 2.7
-- pyOpenSSL
-- python-lxml
+### Prepare Inventory Variables
+TODO: can we skip this step by putting these variables into the repo or is there sensitive info? ideally put them in the test directory
 
-----
+```
+wget http://haste.usersys.redhat.com/raw/uziredenic -O inventory/dynamic/gcp/group_vars/all/vars.yaml
```

Comment:
Lets include example files in this repo too

Reply:
if this is meant only for internal then pls ignore my comment, otherwise pls note the above url doesn't work for a non RH chap like me ;)

Reply:
The variables are now included in the repo in  test/gcp/examples/group_vars/ with instructions in the README to move them into the inventory group_vars. Ideally I would like for these vars to be automatically loaded by the playbook in test, but this seems fine for a quick solution.